### PR TITLE
 Migrating map-layers widget from core repo

### PR DESCRIPTION
**Pull request by mdastous-bentley**

### CODE REVIEW

File: packages/itwin/map-layers/config/rush-project.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (aruniverse) [inline packages/itwin/map-layers/config/rush-project.json:1]: can remove this file

File: packages/itwin/map-layers/package.json [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    "cover": "nyc npm -s test",
+    "docs": "",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "pseudolocalize": "betools pseudolocalize --englishDir=./src/public/locales/en --out=./src/public/locales/en-PSEUDO",
+    "test": "mocha \"./lib/cjs/test/**/*.test.js\"",
+    "rebuild": "npm run -s clean && npm run -s build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/itwinjs-core/tree/master/extensions/map-layers"
```
Comment (aruniverse) [inline packages/itwin/map-layers/package.json:27]: this needs to reflect new repo

File: packages/itwin/map-layers/package.json [hunk 3]
```diff
@@ -0,0 +1,150 @@
+{
+  "name": "@itwin/map-layers",
+  "version": "4.0.0",
```
Comment (aruniverse) [inline packages/itwin/map-layers/package.json:3]: make this a pre-release

File: rush.json [hunk 4]
```diff
@@ -66,6 +66,11 @@
       "packageName": "@itwin/infrastructure-iot-react",
       "projectFolder": "packages/itwin/infrastructure-iot",
       "versionPolicyName": "infrastructure-iot"
-    }
+    },
+    {
+      "packageName": "@itwin/map-layers",
+      "projectFolder": "packages/itwin/map-layers",
+      "versionPolicyName": "itwin-map-layers"
```
Comment (aruniverse) [inline rush.json:73]: need to add this to version-policies.json

File: packages/itwin/map-layers/package.json (hunk 3 above)
Comment (mdastous-bentley) [reply packages/itwin/map-layers/package.json:3]: Sorry I dont understand what you mean, how can it be made pre-release?  Should I add a '-devXX' suffix?

Comment (aruniverse) [reply packages/itwin/map-layers/package.json:3]: actually lets hold off on putting a version for this pkg, lets chat next week about this

**Review by ben-polinsky — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]

**Pull request merged**